smogv2.py

```diff
@@ -108,8 +119,30 @@ def perlin_noise(world, scale=600.0, octaves=6, persistence=0.5, lacunarity=2.0)
 
 
 if __name__ == "__main__":
-    path_to_json = sys.argv[1]
-    save_dir = sys.argv[2]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--json", "-j", type=str)
+    parser.add_argument("--save_dir", "-s", type=str, default="")
+    parser.add_argument("--comet", "-c", action="store_true", default=False)
+    parser.add_argument("--no_pert", action="store_true", default=False)
+    args = parser.parse_args()
+
+    path_to_json = Path(args.json)
+
+    assert path_to_json.exists()
+
+    if args.save_dir:
+        save_dir = Path(args.save_dir)
+        save_dir.mkdir(parents=True, exist_ok=True)
+    else:
+        save_dir = None
+        assert parser.comet, "Specify a --save_dir or use --comet"
+
+    use_perlin_perturbation = not args.no_pert
```

Comment:
ah ok I see now. Well I still think we should have no_pert = True as the default

Reply:
@vict0rsch